cassandra-4/src/main/java/io/debezium/connector/cassandra/Cassandra4CommitLogParserBase.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.debezium.connector.cassandra;
+
+import java.util.List;
+import java.util.Set;
+
+import org.apache.cassandra.db.commitlog.CommitLogPosition;
+import org.apache.cassandra.db.commitlog.CommitLogReader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.DebeziumException;
+import io.debezium.connector.base.ChangeEventQueue;
+import io.debezium.connector.cassandra.exceptions.CassandraConnectorTaskException;
+
+public abstract class Cassandra4CommitLogParserBase {
```

Comment:
I would rather call this `AbstractCassandra4CommitLogParser`

Reply:
done